digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -197,6 +197,17 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
     }
   }
 
+  post("/roadlinks/roadaddress/project/sendToTR"){
+   try{
+     val projectID = parsedBody.extract[projectID]
+    //TODO call getRoadAddressChangesAndSendToTR(projectID). if success change projectstatus to "sent to TR"
```

Comment:
I think it would be easier to add this on Viite-380 task, when we have both 375 and 64 on master.